ToolBox/README.md

```diff
@@ -1,4 +1,4 @@
-Lean Data ToolBox 
+![alt tag](https://cdn.quantconnect.com/web/i/20180601-1615-lean-logo-small.png)  Data ToolBox
```

Comment:
Did we lose the `Lean` prefix here?

Lean Data Toolbox => became => Data Toolbox ?

Reply:
I decided to lose the `Lean` since it's in the logo I added. Do you think I should add it?

Reply:
Let's keep the Lean name, or can make it an alt text if possible -- just thinking of when the logo can't be rendered

Reply:
done, readded `Lean` ;)